corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -401,6 +400,53 @@ hqDefine("cloudcare/js/form_entry/entrycontrols_full", function () {
     MultiSelectEntry.prototype = Object.create(EntryArrayAnswer.prototype);
     MultiSelectEntry.prototype.constructor = EntryArrayAnswer;
 
+    /**
+     * Represents a select2 for multiple options
+     */
+    function MultiDropdownEntry(question, options) {
+        var self = this;
+        MultiSelectEntry.call(this, question, options);
+        self.templateType = 'multidropdown';
+        self.placeholderText = gettext('Please choose an item');
+
+        self.helpText = function () {
+            return "";
+        };
+
+        self.options = ko.computed(function () {
+            return _.map(question.choices(), function (choice, idx) {
+                return {
+                    text: choice,
+                    id: idx + 1,
+                };
+            });
+        });
+
+        self.renderSelect2 = function () {
+            var $input = $('#' + self.entryId);
+            $input.select2(_.extend({
+                allowClear: true,
+                placeholder: self.placeholderText,
+                multiple: true,
```

Comment:
I don't think the `multiple: true` is necessary, select2 should respect the `multiple` attribute in the HTML.